src/core/annotation.js

```diff
@@ -505,26 +502,19 @@ class Annotation {
   _isPrintable(flags) {
     return (
       this._hasFlag(flags, AnnotationFlag.PRINT) &&
+      !this._hasFlag(flags, AnnotationFlag.HIDDEN) &&
```

Comment:
Historically we've had bugs related to Annotation-visiblity and various flags.

Note that this code applies *to all* Annotations, and the code below (i.e. the one setting e.g. `data.display`) *only* applies to WidgetAnnotations, hence please explain why these changes are generally correct/safe!